docs/user/using-an-OIDC-provider.md

```diff
@@ -67,6 +146,8 @@ In the case of Google we can use an OAuth 2.0 client ID. You can find more infor
 
 ### VMware Cloud Services
 
+Note: there is a [more detailed guide on how to use VMware Cloud Services in this document](../step-by-step/kubeapps-on-tkg/step-1.md#step-12-configure-an-oidc-provider).
```

Comment:
Up to you if you think it's worth just replacing this whole section with the link to your improved docs there?